### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by rdgreene** [rdgreene/MNIST#11]
@ginzsa-push, I need numerical results for the following:

Hyperparameter Tuning (i.e. Cross-Validtion):
  average **training & validation classification  error** for each different kernel choice 
  average **training & validation  classification error** for each combination of box-constraint and kernel scale

Final Trained Model:
  classification error on **training dataset**
  classification testing error on  **testing dataset**
  **number** of digits in testing dataset **correctly classified**

Also as discussed yesterday:
  **yHat** vector for **testing** dataset
  **score** matrix for **testing** dataset

Let me know if any questions 💖

**Comment by ginzsa-push** [comment]
svm_fit_test_data.mat   >> contains ClassAcc, Class Error (in testing
dataSet), labelOut = yhat and score

**Comment by ginzsa-push** [comment]
classification testing error on testing dataset = 0.0189  Accuracy = 0.9811

**Comment by ginzsa-push** [comment]
average training & validation classification error for each combination of box-constraint and kernel scale

https://docs.google.com/spreadsheets/d/1mUU1vMiirQAV8Kl5FXwWZx1N4NczJZ82sAYPu-0KJfo/edit#gid=391326543
sheet 4 = Validation errors
sheet 5 =  Training errors

**Comment by rdgreene** [comment]
Nice, thanks!
